charts/sophora-image-ai/values.yaml

```diff
@@ -0,0 +1,58 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repository: docker.subshell.com/sophora/imageai
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "4.0.0"
+
+imagePullSecrets: []
+
+javaOptions: -XX:+ExitOnOutOfMemoryError -XX:+PerfDisableSharedMem
```

Comment:
Whenever the value contains a `:` I'd surround the value with quotation marks.

Reply:
I usually don't. To a compliant YAML parser, this should be a regular string.